docs/otlp/getting-started-aspnetcore/Program.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Diagnostics;
+using OpenTelemetry;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Resources;
+using OpenTelemetry.Trace;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Clear default log providers added by the host.
+builder.Logging.ClearProviders();
+
+// Configure OpenTelemetry with logs, metrics, & tracing and auto-start.
+builder.Services.AddOpenTelemetry()
+    .ConfigureResource(resource => resource
+        .AddService(serviceName: builder.Environment.ApplicationName))
+    .WithTracing(tracing => tracing.AddAspNetCoreInstrumentation())
+    .WithMetrics(metrics => metrics.AddAspNetCoreInstrumentation())
+    .UseOtlpExporter();
+
+// Set OpenTelemetry metrics to use delta temporality.
+builder.Services.Configure<MetricReaderOptions>(
+    o => o.TemporalityPreference = MetricReaderTemporalityPreference.Delta);
```

Comment:
Does AspireDashboard tool handles delta/cumulative correctly? If it does, then we are good. If it is designed to handle delta only (many backends are!, except prometheus), then this will become inevitable:(

Reply:
Seems to work fine in cumulative with Aspire so I removed it. @JamesNK Can you confirm what is supported?

@noahfalk I agree with you it seems too complex for a "hello world" but I felt best practice it might be best to show it 🤷 Kind of a pit of failure the way the spec has set this part designed IMO. User has to a) understand different modes exist, b) figure out what they need, and c) figure out what the default is and how to change the value if needed. Ouch!

Reply:
I don't know. I know Aspire works with metrics + default settings. I haven't tried what happens with delta only.

Reply:
If you can avoid having it that seems preferable - it sounds like it is avoidable with Aspire dashboard at least. If you do need it (perhaps you'll want to demo some other OTLP viewers that require it?) then its probably worth some additional explanatory text to help users understand why its there.

> Kind of a pit of failure the way the spec has set this part designed IMO

Yeah, that definitely feels unfortunate to have designed the protocol where different tools require different options and there is no auto-negotiation of the setting inside the communication channel.